components/empty-state/empty-state-illustrated-link.js

```diff
@@ -0,0 +1,62 @@
+import '../link/link.js';
+import { bodyCompactStyles, heading2Styles  } from '../typography/styles.js';
+import { emptyStateIllustratedStyles, emptyStateStyles } from './empty-state-styles.js';
+import { html, LitElement } from 'lit';
+import { getIllustration } from './getIllustration.js';
+import { RtlMixin } from '../../mixins/rtl-mixin.js';
+import { runAsync } from '../../directives/run-async/run-async.js';
+
+/**
+ * The `d2l-empty-state-illustrated-link` component is an empty state component that displays an illustration and action link. The illustration property can be set to use one of the preset illustrations or a custom SVG illustration can be added in the default slot.
+ * @slot - Custom SVG content if illustration property is not set
+ */
+class EmptyStateIllustratedLink extends RtlMixin(LitElement) {
+
+	static get properties() {
+		return {
+			/**
+			  * Optional: The action URL or URL fragment of the link
+			  * @type {String}
+			  */
+			actionHref: { type: String, attribute: 'action-href' },
+			/**
+			 * Optional: The action text to be used in the link
+			 * @type {String}
+			 */
+			actionText: { type: String, attribute: 'action-text' },
+			/**
+			 * REQUIRED: A description giving details about the empty state
+			 * @type {String}
+			 */
+			description: { type: String },
+			/**
+			 * Optional: The name of the preset image you would like to display in the component
+			 * @type {String}
+			 */
+			illustration: { type: String },
+			/**
+			 * REQUIRED: A title for the empty state
+			 * @type {String}
+			 */
+			title: { type: String },
```

Comment:
So `title` is actually a reserved HTML attribute name for showing a native browser tooltip on-hover -- I'm guessing if you hover over the component you'll see this as a tooltip, which isn't what you intended. :)

I'd recommend `title-text` as an alternate name, which we've used elsewhere.